src/editor-previewer/editor-previewer.component.html

```diff
@@ -4,6 +4,9 @@
     <tab *ngFor="let preview of previews trackBy:trackByFunction" [heading]="preview.name">
       <div [ngSwitch]="preview.type">
         <div *ngSwitchCase="'html'" class="preview-container">
+          <template tabHeading>
+            <span (click)="openPreviewInNewWindow(preview.url)">&#10138;</span>
```

Comment:
Can you make this an `<a target="_blank" [href]="preview.url">&#10138;</a>` and `text-decoration: none` in css?. In this way it will have hand mouse icon on hover and you don't have to window.open since `anchor` has `href`